config/settings/base.py

```diff
@@ -348,6 +348,12 @@ def env(name, default=NoDefaultValue, type_=str):
 }
 
 
+# SF client settings:
+SF_CALLBACK_URL = env("SF_CALLBACK_URL", default=None)
+SF_CLIENT_KEY = env("SF_CLIENT_KEY", default=None)
+SF_CLIENT_ID = env("SF_CALLBACK_URL", default=None)
+SF_CLIENT_SECRET = env("SF_CLIENT_SECRET", default=None)
```

Comment:
@wlonk Can we update the docs (and presumably `env.example`) with instructions for setting these? I'd assume the `populate_social_apps` command could just use these env vars instead of passed-in args, for SF at least.

Reply:
@jgerigmeyer Let me know if this latest push satisfies your request.